src/components/Tweet.js

```diff
@@ -1,36 +1,25 @@
-function Tweet() {
+import Actions from "./Actions";
+import Message from "./Message";
+import ProfileImage from "./ProfileImage";
+import Timestamp from "./Timestamp";
+import User from "./User";
+
+function Tweet({ tweet }) {
```

Comment:
you could try further destructuring tweet to get image, timestamp, user and message from it :)